recipe/meta.yaml

```
@@ -25,6 +25,7 @@ requirements:
     - {{ compiler('fortran') }}  # [not win]
     - {{ compiler('c') }}
     - cmake  # [win]
+    - make
```

Should this not be
```suggestion
    - make  # [not win]
```
? The CI only fails on linux_aarch64.

---

Yes 👍 

Make is already installed in the other unix images but it's still better to get make from conda-forge rather than relying on the host system.